### PR TITLE
Change role name to list of role arns

### PR DESCRIPTION
**Pull request by goruha**
## What
* Change `var.membership` and `var.role_name` with `var.role_arns`

## Why
* To change pattern grant access to assume roles

### CODE REVIEW

**Comment by goruha** [comment]
@osterman choose one.
There is 2 concepts 
* Group that allow assuming different roles from one account
* Group that allow assuming different roles from different accounts

**Comment by osterman** [comment]
Good point.

File: example/main.tf [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Provision group access to data account
+module "organization_access_group_data" {
+  source      = "../"
+  enabled     = "true"
+  namespace   = "cp"
```
Comment (aknysh) [inline example/main.tf]:
```suggestion
  namespace   = "eg"
```

Let's not use `cp` for the namespace.
People copy the examples verbatim and get errors that some global AWS resources with that name already exist

File: output.tf [hunk 2]
```diff
@@ -28,7 +28,7 @@ output "policy_id" {
   description = "The policy ID"
 }
 
-output "switchrole_url" {
-  description = "URL to the IAM console to switch to the role"
-  value       = "${local.enabled ? format(var.switchrole_url, var.member_account_id, var.role_name, module.label.id) : ""}"
+output "switchrole_urls" {
+  description = "List of URL to the IAM console to switch to the roles"
+  value       = ["${formatlist(var.switchrole_url, null_resource.role.*.triggers.account_id, null_resource.role.*.triggers.role_name, null_resource.role.*.triggers.alias)}"]
```
Comment (aknysh) [inline output.tf]:
```suggestion
  value       = ["${formatlist(var.switchrole_url_template, null_resource.role.*.triggers.account_id, null_resource.role.*.triggers.role_name, null_resource.role.*.triggers.alias)}"]
```

This is already a template

File: variables.tf [hunk 3]
```diff
@@ -41,7 +36,7 @@ variable "name" {
   description = "Name  (e.g. `app` or `cluster`)"
 }
 
-variable "switchrole_url" {
+variable "switchrole_url_template" {
   type        = "string"
   description = "URL to the IAM console to switch to a role"
```
Comment (aknysh) [inline variables.tf]:
```suggestion
  description = "URL template for the IAM console to switch to the roles"
```

**Pull request merged**